mappings/net/minecraft/recipe/Ingredient.mapping

```diff
@@ -32,8 +34,8 @@ CLASS net/minecraft/class_1856 net/minecraft/recipe/Ingredient
 		ARG 0 tag
 	METHOD method_8107 entryFromJson (Lcom/google/gson/JsonObject;)Lnet/minecraft/class_1856$class_1859;
 		ARG 0 json
-	METHOD test (Ljava/lang/Object;)Z
-		ARG 1 stack
+	METHOD test testStack (Ljava/lang/Object;)Z
```

Comment:
No, you broke the method that properly overrides the method `boolean test(Object)` in `Predicate`.